nifi-marklogic-processors/src/main/java/org/apache/nifi/marklogic/processor/ApplyTransformMarkLogic.java

```diff
@@ -53,19 +54,39 @@
 public class ApplyTransformMarkLogic extends QueryMarkLogic {
     public static final PropertyDescriptor APPLY_RESULT_TYPE = new PropertyDescriptor.Builder()
         .name("Apply Result Type").displayName("Apply Result Type").defaultValue(ApplyResultTypes.REPLACE.getValue())
-        .description("Whether to REPLACE each document with the result of the transform, or run the transform with each document as input, but IGNORE the result.").required(true)
+        .description("Whether to REPLACE each document with the result of the transform, or run the transform with each document "
```

Comment:
This will need some formatting - as shown in the screenshot, it's difficult to read the description. We'll need more work done than just copying/pasting the text from the docs into here, it needs more formatting.

![image](https://user-images.githubusercontent.com/5026193/201746841-293b707e-f50c-46c9-965e-9af8695fc23c.png)